### PR TITLE
refactor(clients): each defines it's kind, handler and required keys

### LINKED ISSUE #55: Move client kinds and required keys from core to server
Related to notify-watcher/core#19

### PR DESCRIPTION
**Pull request by negebauer**
Related to notify-watcher/core#21
Closes #55

### CODE REVIEW

File: src/watchers/index.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ async function setUpWatchers() {
   } = loadWatchers(WATCHERS_PATH);
 
   const minuteWatchersCronJob = new CronJob(
-    // '0,5,10,15,20,25,30,35,40,45,50,55 * * * * *', // for dev
-    '0 * * * * *',
+    '0,5,10,15,20,25,30,35,40,45,50,55 * * * * *', // for dev
```
Comment (sasalatart) [inline src/watchers/index.js]: Removing this comment is ok?

File: src/watchers/run-watchers.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const LOCAL_ENV = {
   watcherIteration: false,
   watcherAuthRun: false,
   watcherNoAuthRun: false,
-  alwaysRunDayWatcher: false,
+  alwaysRunDayWatcher: true,
```
Comment (sasalatart) [inline src/watchers/run-watchers.js]: Is this part of this PR? (It's np if it's not, but was just wandering, as it does not have to do with the title haha)

Comment (negebauer) [reply src/watchers/run-watchers.js]: missed that revert

File: src/watchers/index.js (hunk 1 above)
Comment (negebauer) [reply src/watchers/index.js]: missed that revert

**Pull request merged**